visual-js/visual-wdio/src/SauceVisualService.ts

```diff
@@ -163,86 +147,65 @@ ${url.padStart(100, ' ')}
 `;
 }
 
-function splitIgnorables(
-  a: Array<IgnoreRegion | WdioElement | WdioElement[]>,
-): { elements: WdioElement[]; regions: IgnoreRegion[] } {
-  const elements: WdioElement[] = [];
-  const regions: IgnoreRegion[] = [];
-  for (let i = 0; i < a.length; i++) {
-    const x = a[i];
-
-    if (Array.isArray(x)) {
-      elements.concat(...x);
-    } else if (isWdioElement(x)) {
-      elements.push(x);
-    } else {
-      try {
-        const region = validateIgnoreRegion(x);
-        regions.push(region);
-      } catch (e) {
-        throw new Error(
-          `Ignored element options.ignore[${i}] is not valid: ${e}`,
-        );
-      }
-    }
+const getDiffingOptions = (
+  region: RegionType,
+): DiffingOptionsIn | undefined => {
+  const { element: __, ...rest } = region;
+  if ('enableOnly' in rest || 'disableOnly' in rest) {
+    return selectiveRegionOptionsToDiffingOptions(rest);
```

Comment:
Is there specific reason why you want to let the backend handle the default instead of `selectiveRegionOptionsToDiffingOptions`?

If the behaviour is not satisfactory, we should change `selectiveRegionOptionsToDiffingOptions` directly and not wrap it.

Reply:
It's less for us to maintain long term if we let the backend handle as much of the defaults as we can.